serviceregistry-server/src/main/java/no/difi/meldingsutveksling/serviceregistry/model/ServiceIdentifier.java

```diff
@@ -15,5 +15,7 @@
     /**
      * Identifies DIFI sikker digital post aka. Digital Post Innbygger
      */
-    DPI
+    DPI,
+    DPE_innsyn,
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS00115)